SpatialGDK/Source/SpatialGDKServices/Public/SpatialGDKServicesConstants.h

```diff
@@ -32,7 +32,7 @@ namespace SpatialGDKServicesConstants
 	const FString SpatialOSConfigFileName = TEXT("spatialos.json");
 	const FString ChinaEnvironmentArgument = TEXT(" --environment=cn-production");
 
-	const FString SpatialOSRuntimePinnedStandardVersion         = TEXT("0.4.0");
+	const FString SpatialOSRuntimePinnedStandardVersion         = TEXT("0.4.1");
```

Comment:
Can we please confirm that there are no issues with the replace a worker flow with this release? Start / stop for a single deploy works for like 10 tries etc.

Reply:
I don't think that should have changed, but I can try.

Reply:
Tried starting/restarting, couldn't see any issues.
Of course, replace a worker in the cloud is still not supported.